code/default/gae_proxy/local/proxy.py

```diff
@@ -118,16 +118,21 @@ def main(args):
     CertUtil.init_ca()
 
     allow_remote = args.get("allow_remote", 0)
-    if allow_remote:
-        listen_ip = "0.0.0.0"
+
+    listen_ips = front.config.listen_ip
+    if isinstance(listen_ip, str):
```

Comment:
`listen_ip`应该是`listen_ips`，另外这个的类型不是str而是unicode……
```suggestion
    if not isinstance(listen_ips, list):
```

Reply:
unicode 吗，我再看看。。。

Reply:
换成 basestring 了